azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/handlers/runtime/BaseRuntimeHandler.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.handlers.runtime;
+
+import com.microsoft.azure.management.appservice.AppServicePlan;
+import com.microsoft.azure.management.appservice.WebAppBase;
+import com.microsoft.azure.maven.handlers.RuntimeHandler;
+
+public class BaseRuntimeHandler<T extends WebAppBase> implements RuntimeHandler<T> {
+    @Override
+    public WebAppBase.DefinitionStages.WithCreate defineAppWithRuntime() throws Exception {
```

Comment:
What does this class do? I was about to suggest making this an interface since its implementation is almost empty. But you already have RuntimeHandler interface. What is the point to have a man in the middle which does nothing?

Reply:
I'll move the common part from webapp BaseRuntimeHandler to this class, including the common attributes and constructor, but it includes much code and test cases changes, will do it in next pr.